src/confluent_kafka/admin/__init__.py

```diff
@@ -247,20 +247,21 @@ def _make_acls_result(f, futmap):
     def _make_user_scram_credentials_result(f, futmap):
```

Comment:
Since we are given a dict and we have to only set the result, so no change to alter would be there. Also __all__ is checked when futmap is empty , incase of alter we would always have non empty futmap so it does not affect the alter workflow.